jenkins_pipelines/environments/common/pipeline-pull-request-light.groovy

```diff
@@ -0,0 +1,208 @@
+def run(params) {
```

Comment:
I would guess this "-light" pipeline is just temporary, and we will replace the content of this one inside "pipeline-pull-request.groovy" once it's stable?
Otherwise, we will have an extra pipeline, and a older one not anymore in use.